J.Coate_H.Nguyen_Case_Study_1.Rmd

```diff
@@ -156,6 +168,11 @@ brew.beer[which.max(brew.beer$ABV),]
 ```
 ### The state with the highest ABV is Colorado at 12.8%.
 
+## Huy's comment: 
+It's the maximum ABV, not the median. then we use this code
+brew.beer
+brew.beer[which.max(brew.beer$ABV),]
+
```

Comment:
Right, that's what I said below. Does your code produce the same result as mine? If so we can just leave what I wrote because mine provides more information like state, brewery, etc.

Reply:
I agree with you